### PR TITLE
BDOG-2587: correct accessibility issues

### PR DESCRIPTION
**Pull request by victorarbuesmallada**
Just the fixes for the (hopefully) last 27 accessibility issues.

### CODE REVIEW

File: app/views/partials/dependency_section.scala.html [hunk 1]
```diff
@@ -65,8 +66,8 @@
         </span>
         <span id="@{dependency.name}-@{rootId}-bobbyrule" class="col-xs-2">
         @dependency.versionState match {
-            case Some(VersionState.BobbyRulePending (violation)) => { <a id="see-pending-bobby-rule-@dependency.group:@dependency.name" href="@{appRoutes.BobbyExplorerController.list()}#@{dependency.group}:@{dependency.name}:@{violation.range}">See rule</a> }
-            case Some(VersionState.BobbyRuleViolated(violation)) => { <a href="@{appRoutes.BobbyExplorerController.list()}#@{dependency.group}:@{dependency.name}:@{violation.range}">See rule</a> }
+            case Some(VersionState.BobbyRulePending (violation)) => { <a id="see-pending-bobby-rule-@dependency.group:@dependency.name" href="@{appRoutes.BobbyExplorerController.list()}#@{dependency.group}:@{dependency.name}:@{UrlUtils.encodeQueryParam(violation.range.range)}">See rule</a> }
```
Comment (victorarbuesmallada) [inline app/views/partials/dependency_section.scala.html:69]: jfyi
```
Bad value “/bobbyrules#uk.gov.hmrc:reactivemongo-test:[4.0.0,4.4.0]” for attribute “href” on element “a”: Illegal character in fragment: “[” is not allowed.
```

File: app/views/vulnerabilities/VulnerabilitiesListPage.scala.html [hunk 2]
```diff
@@ -108,29 +108,29 @@ <h1>Vulnerabilities</h1>
...
-        <td role="cell" class="curation-status hand-pointer" data-toggle="collapse" data-target="#collapsible-area-@summary.distinctVulnerability.id" aria-expanded="false" aria-controls="collapsible-area-@summary.distinctVulnerability.id"><div>@summary.distinctVulnerability.curationStatus.map(_.display).getOrElse("")</div></td>
-        <td role="cell" class="score hand-pointer text-right" data-toggle="collapse" data-target="#collapsible-area-@summary.distinctVulnerability.id" aria-expanded="false" aria-controls="collapsible-area-@summary.distinctVulnerability.id">@summary.distinctVulnerability.score.getOrElse("")</td>
-        <td role="cell" class="services text-right hand-pointer" data-toggle="collapse" data-target="#collapsible-area-@summary.distinctVulnerability.id" aria-expanded="false" aria-controls="collapsible-area-@summary.distinctVulnerability.id"><div>@summary.occurrences.groupBy(_.service).size</div></td>
+        <td class="accordion-toggle collapsed hand-pointer " data-toggle="collapse" data-target="#collapsible-area-@summary.distinctVulnerability.id" aria-controls="collapsible-area-@summary.distinctVulnerability.id"/>
```
Comment (victorarbuesmallada) [inline app/views/vulnerabilities/VulnerabilitiesListPage.scala.html:111]:
```
Attribute “aria-expanded” not allowed on element “td” at this point.
```
```
The “role” attribute must not be used on a “td” element which has a “table” ancestor with no “role” attribute, or with a “role” attribute whose value is “table”, “grid”, or “treegrid”.
```

File: app/views/partials/dependency_section.scala.html (hunk 1 above)
Comment (colin-lamed) [reply app/views/partials/dependency_section.scala.html:69]: presumably these ids were added for auditing, and have no impact on acceptance tests

**Review by colin-lamed — APPROVED** [review]

File: app/views/partials/dependency_section.scala.html (hunk 1 above)
Comment (victorarbuesmallada) [reply app/views/partials/dependency_section.scala.html:69]: yeah, acceptance tests are passing :)

**Pull request merged**